### PR TITLE
TW-995: 0xOptimal. Finished adding checkbox

### PR DESCRIPTION
**Pull request by herkoss**

### CODE REVIEW

File: src/screens/create-new-wallet/AnalyticsField.tsx [hunk 1]
```diff
@@ -11,9 +11,7 @@ import { setIsAnalyticsEnabled } from 'src/store/settings/settings-actions';
 import { formatSize } from 'src/styles/format-size';
 import { useSetPasswordScreensCommonStyles } from 'src/styles/set-password-screens-common-styles';
 
-import { CreateNewWalletSelectors } from './create-new-wallet.selectors';
-
-export const AnalyticsField: FC<{ enabled: boolean }> = ({ enabled }) => {
+export const AnalyticsField: FC<{ enabled: boolean; testID?: string }> = ({ enabled, testID }) => {
```
Comment (lendihop) [inline src/screens/create-new-wallet/AnalyticsField.tsx:14]: interface Props { 
enabled: boolean; 
testID?: string 
}

File: src/screens/create-new-wallet/view-ads-field.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import { useDispatch } from 'react-redux';
+
+import { CheckboxLabel } from 'src/components/checkbox-description/checkbox-label';
+import { Divider } from 'src/components/divider/divider';
+import { FormCheckbox } from 'src/form/form-checkbox';
+import { setIsAnalyticsEnabled } from 'src/store/settings/settings-actions';
+import { formatSize } from 'src/styles/format-size';
+import { useSetPasswordScreensCommonStyles } from 'src/styles/set-password-screens-common-styles';
+
+export const ViewAdsField: FC<{ enabled: boolean; testID?: string }> = ({ enabled, testID }) => {
```
Comment (lendihop) [inline src/screens/create-new-wallet/view-ads-field.tsx:12]: the same interface here

**Review by lendihop — APPROVED** [review]

**Review by lendihop — APPROVED** [review]

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15_902

**Review by vladvn1905 — APPROVED** [review]
ios done build: 1.15_902

**Pull request merged**